main/zoe/api/zoe-helpers.md

```diff
@@ -236,3 +283,62 @@ const sell = seat => {
   return 'Trade Successful';
 };
 ```
+ 
+## depositToSeat(zcf, recipientSeat, amounts, payments)
+- `zcf` `{ContractFacet }`
+- `recipientSeat` `{ZCFSeat}`
+- `amounts` `{AmountKeywordRecord}`
+- `payments` `{PaymentPKeywordRecord}`
+- Returns: `{Promise<string>}`
+Deposit payments such that their amounts are reallocated to a seat.
+The `amounts` and `payments` records must have corresponding
+keywords.
+
+If the seat has exited, aborts with the message `The recipientSeat cannot have exited.`
+
```

Comment:
I don't think this is true. It's the default message in trade

Reply:
Lines 349-50 in zoe/src/contractSupport/zoeHelpers.js

export async function depositToSeat(zcf, recipientSeat, amounts, payments) {
  assert(!recipientSeat.hasExited(), 'The recipientSeat cannot have exited.');

Reply:
Ah good point! Thanks, you were right.